src/libraries/System.Security.Cryptography.Primitives/ref/System.Security.Cryptography.Primitives.cs

```diff
@@ -164,7 +164,7 @@ public partial interface ICryptoTransform : System.IDisposable
         bool CanTransformMultipleBlocks { get; }
         int InputBlockSize { get; }
         int OutputBlockSize { get; }
-        int TransformBlock(byte[] inputBuffer, int inputOffset, int inputCount, byte[]? outputBuffer, int outputOffset);
```

Comment:
By the review comment https://github.com/dotnet/corefx/pull/42298#discussion_r341752504 `?` removed from `byte[]? outputBuffer` in source, but my bad 😛, forgot to remove it from ref assembly

Reply:
This is one of the downsides of not having validation in between src and ref for this yet 😢